File: 2798_number_of_employees_who_met_the_target/solution.c
// LeetCode 2798 - Number of Employees Who Met the Target
// https://leetcode.com/problems/number-of-employees-who-met-the-target/

int numberOfEmployeesWhoMetTarget(int* hours, int hoursSize, int target) {
    int ans = 0;
    for (int i = 0; i < hoursSize; i++) {
        if (hours[i] >= target) ans++;
    }
    return ans;
}

File: 2798_number_of_employees_who_met_the_target/solution.rb
# LeetCode 2798 - Number of Employees Who Met the Target
# https://leetcode.com/problems/number-of-employees-who-met-the-target/

# @param {Integer[]} hours
# @param {Integer} target
# @return {Integer}
def number_of_employees_who_met_target(hours, target)
  hours.count { |h| h >= target }
end
